<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmDeferCloseCalDlg</class>
 <widget class="QDialog" name="frmDeferCloseCalDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>平仓试算</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>631</width>
     <height>301</height>
    </rect>
   </property>
   <widget class="QSpinBox" name="spinBox_num">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>58</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_down_val">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>98</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>空仓值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>198</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>空仓剩余数量：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_up_count">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>168</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_type">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>58</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>148</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>平仓盈亏：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_up_val">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>103</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>多仓值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>平仓类型：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>228</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
        border-radius: 4px;
        border: none;
        width:  50px;
        height: 25px;
		background: #2a3a57;
}
QPushButton:enabled {
        background: #2a3a57;
        color: white;
}
QPushButton:enabled:hover{
        background: #227dc3;
}

QPushButton:enabled:pressed{
        background: #227dc3;
}</string>
    </property>
    <property name="text">
     <string>试算</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>343</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>平仓数量：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_code">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>28</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_close_gain">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>148</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_price">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>28</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>133</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>平仓手续费：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>103</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 75 11pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>多仓：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>178</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>平仓后可用金额：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_close_canuse">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>178</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>343</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>平仓价格：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>168</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>多仓剩余数量：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_down_count">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>198</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>97</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 0);
font: 75 11pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>空仓：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>合约代码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_close_fee">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
